setup.py

```diff
@@ -21,7 +21,7 @@
     ],
     setup_requires=["wheel"],
     tests_require=["pytest"],
-    extras_require={"influence": ["torch"]},
+    extras_require={"influence": ["torch>=2.0.0"], "cupy": ["cupy-cuda11x>=12.1.0"]},
```

Comment:
cupy is only used with influence functions, right? What would be the disadvantage of adding it to the influence extras instead of creating a separate one just for it?

Reply:
On mac, there is no wheel cupy-cuda11x. If we include it into the influence extra, then no mac user could install the influence extra

Reply:
Alright, we can leave it for now.
I will create a separate issue to consider using [environment markers](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#environment-markers) for this.